src/control/drpc/constants.go

```diff
@@ -0,0 +1,14 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package drpc
```

Comment:
No reason to put this in drpc if it's not being pulled over from C code.

An aside, I think we should create a package for DAOS/CART lib constants in the near future. Putting them in drpc is starting to get ugly.

Reply:
Yeah... I started there because I assumed that I'd be able to get at it. I agree that we probably need a `lib/daos` package to more cleanly organize things.